python/src/config/__init__.py

```diff
@@ -55,9 +57,13 @@ def get_config():
         config.AWS_ACCOUNT_ID = "000000000000"
         config.BOTO_RESOURCE_KWARGS["aws_access_key_id"] = "my-key"
         config.BOTO_RESOURCE_KWARGS["aws_secret_access_key"] = "my-secret-key"
+        global_sdk_config.set_sdk_enabled(False)
```

Comment:
What does sdk_enabled do? (besides enabling sdk) What is sdk?

Reply:
Enables XRay.